framework/src/org/checkerframework/framework/util/AnnotatedTypes.java

```diff
@@ -1259,19 +1259,20 @@ public AnnotatedTypeMirror leastUpperBound(AnnotatedTypeMirror type,
      * Given a list of types representing the formal parameters to a method, get the parameter type
      * expect at the indexth position (unwrapping var args if necessary).
      *
+     * @param hasVarArg whether or not that last parameter is a vararg
```

Comment:
The 'that last parameter' reference wasn't clear to me - it depends on the parenthesized part of the body of the javadoc. I would make the `@param` descriptions as independent as possible.